packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -483,6 +420,7 @@ class _ReorderableListViewState extends State<ReorderableListView> {
         SliverPadding(
           padding: listPadding,
           sliver: SliverReorderableList(
+            scrollDirection: widget.scrollDirection,
```

Comment:
What if this conflicts with the scroll direction provided by the constraints during layout?